docs/src/main/sphinx/release/release-405.md

```diff
@@ -0,0 +1,87 @@
+# Release 405 (23 Nov 2022)
+
+## General
+
+* Improve performance of queries which compare `date` columns with
+  `timestamp with time zone` literals. ({issue}`5798`)
+* Improve performance and resource utilization of `INSERT` queries into
+  partitioned and bucketed tables. ({issue}`14718`)
+
+## Security
+
+* Add support for string replacement as part of
+  [impersonation rules](system-file-auth-impersonation-rules). ({issue}`14962`)
+* Fix some `system.metadata` tables showing the names of catalogs which the user
```

Comment:
Fix display of catalogs that the user can not access in some ...

Reply:
I think I like the current wording better... it feels a little more natural.